### ISSUE TITLE
Keyboard input not working in IE Mobile 11 (Windows Phone 8.1)

### ISSUE THREAD

**Issue by kenrick95** [mathquill/mathquill#596]
I'm using WP8.1 & IE Mobile 11 is the only available browser there. 'm here to report that I when I tap at the textbox, the on-screen keyboard did appear and I can tap my keys but the characters did not appear in the textbox.

Maybe related to #559, and maybe this has very low priority too (given the small market share).

Thanks.

**Comment by laughinghan** [comment]
Thanks for reporting this! Do you happen to know the version of MathQuill affected? A few days ago we released a fix for some major typing issues (hadn't repro'd any of #559 but we were hoping they would also be fixed), and we pushed the fix to the homepage within the last day, can you check if you can still repro on the homepage?

**Comment by kenrick95** [comment]
Yeah I just tried it on the homepage and this issue can still be reproduced.

**Comment by laughinghan** [comment]
Rats.

Han

**Comment by kyvaith** [comment]
This is duplicate of https://github.com/mathquill/mathquill/issues/65
